Jenkinsfile

```diff
@@ -5,6 +5,27 @@ elifePipeline({
         commit = elifeGitRevision()
     }
 
+    elifeOnNode(
```

Comment:
the build is a warzone but it will improve once we follow-up with `docker-compose`